frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsStats/FeatureMetricsStats.tsx

```diff
@@ -34,7 +35,9 @@ export const FeatureMetricsStats = ({
             <Grid item xs={12} sm={4}>
                 <article className={styles.item}>
                     <h3 className={styles.title}>Exposure</h3>
-                    <p className={styles.value}>{totalYes}</p>
+                    <p className={styles.value}>
+                        <PrettifyLargeNumber value={totalYes} />
```

Comment:
That's a great point! Maybe @NicolaeUnleash can help us here 👍

Reply:
Had a quick chat with @NicolaeUnleash and for now we believe `FeatureOverviewEnvironmentMetrics` could also be a good candidate, since it shows the same numbers: https://github.com/aneeshrelan/unleash/blob/feature/prettify-large-numbers/frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/FeatureOverviewEnvironmentMetrics/FeatureOverviewEnvironmentMetrics.tsx#L71

Reply:
Noted, will add it there as well